### ISSUE TITLE
Having blank screen after clicking "Advanced Settings" for Date field editing on Content-Type Builder

### ISSUE THREAD

**Issue by raspi0124** [strapi/strapi#18445]
### Required System information

- Node.js version: 18.18.1
- NPM version: 9.8.1
- Strapi version: v4.14.4
- Database: MariaDB 10.11
- Operating system: node:18-alpine (Docker Image)
- Is your project Javascript or Typescript: Javascript

### Describe the bug

I'm having a screen turning to white blank screen after clicking "Advanced Settings" for Date field editing on Content-Type Builder.

### Steps to reproduce the behavior

1. Create random collection type and save.
2. Create Datetime field on Content-Type Builder
3. Set Date time type to Date&Time
4. Click on "Advanced Setting" for setting Datetime field
5. The screen turns to blank white screen
6. The console returns "Uncaught TypeError: Ge.toDate is not a function"

### Code snippets

> 21:22:05.160 EvalError: call to eval() blocked by CSP
> 21:22:05.165 Content-Security-Policy: ページの設定により次のリソースの読み込みをブロックしました: eval (“script-src”)
> Source: ;(function r(e,t=!1){const o="6.0";let i... api::notice.notice
> 21:22:13.744 EvalError: call to eval() blocked by CSP
> 21:22:13.758 Content-Security-Policy: ページの設定により次のリソースの読み込みをブロックしました: eval (“script-src”)
> Source: ;(function n(e){let t=1e3,n=10;function ... api::notice.notice
> 21:25:04.744 TypeError: Ge.toDate is not a function
>     N http://localhost:1337/admin/main.58146bb6.js:1426
>     ji http://localhost:1337/admin/main.58146bb6.js:736
>     _c http://localhost:1337/admin/main.58146bb6.js:738
>     xh http://localhost:1337/admin/main.58146bb6.js:738
>     Qd http://localhost:1337/admin/main.58146bb6.js:738
>     tp http://localhost:1337/admin/main.58146bb6.js:738
>     jc http://localhost:1337/admin/main.58146bb6.js:738
>     vf http://localhost:1337/admin/main.58146bb6.js:738
>     od http://localhost:1337/admin/main.58146bb6.js:738
>     _s http://localhost:1337/admin/main.58146bb6.js:736
>     Bs http://localhost:1337/admin/main.58146bb6.js:738
>     Bs http://localhost:1337/admin/main.58146bb6.js:738
>     Jl http://localhost:1337/admin/main.58146bb6.js:738
>     Od http://localhost:1337/admin/main.58146bb6.js:736
>     onTabClick http://localhost:1337/admin/main.58146bb6.js:2169
>     ze http://localhost:1337/admin/main.58146bb6.js:2169
>     pt http://localhost:1337/admin/main.58146bb6.js:735
>     fe http://localhost:1337/admin/main.58146bb6.js:735
>     Ee http://localhost:1337/admin/main.58146bb6.js:735
>     Yt http://localhost:1337/admin/main.58146bb6.js:735
>     xn http://localhost:1337/admin/main.58146bb6.js:735
>     qr http://localhost:1337/admin/main.58146bb6.js:735
>     Gd http://localhost:1337/admin/main.58146bb6.js:738
>     Yn http://localhost:1337/admin/main.58146bb6.js:735
>     qr http://localhost:1337/admin/main.58146bb6.js:735
>     Gr http://localhost:1337/admin/main.58146bb6.js:735
>     jr http://localhost:1337/admin/main.58146bb6.js:735
> main.58146bb6.js:738:162
> 21:25:04.747 Uncaught TypeError: Ge.toDate is not a function
>     N http://localhost:1337/admin/main.58146bb6.js:1426
>     ji http://localhost:1337/admin/main.58146bb6.js:736
>     _c http://localhost:1337/admin/main.58146bb6.js:738
>     xh http://localhost:1337/admin/main.58146bb6.js:738
>     Qd http://localhost:1337/admin/main.58146bb6.js:738
>     tp http://localhost:1337/admin/main.58146bb6.js:738
>     jc http://localhost:1337/admin/main.58146bb6.js:738
>     vf http://localhost:1337/admin/main.58146bb6.js:738
>     od http://localhost:1337/admin/main.58146bb6.js:738
>     _s http://localhost:1337/admin/main.58146bb6.js:736
>     Bs http://localhost:1337/admin/main.58146bb6.js:738
>     Bs http://localhost:1337/admin/main.58146bb6.js:738
>     Jl http://localhost:1337/admin/main.58146bb6.js:738
>     Od http://localhost:1337/admin/main.58146bb6.js:736
>     onTabClick http://localhost:1337/admin/main.58146bb6.js:2169
>     ze http://localhost:1337/admin/main.58146bb6.js:2169
>     pt http://localhost:1337/admin/main.58146bb6.js:735
>     fe http://localhost:1337/admin/main.58146bb6.js:735
>     Ee http://localhost:1337/admin/main.58146bb6.js:735
>     Yt http://localhost:1337/admin/main.58146bb6.js:735
>     xn http://localhost:1337/admin/main.58146bb6.js:735
>     qr http://localhost:1337/admin/main.58146bb6.js:735
>     Gd http://localhost:1337/admin/main.58146bb6.js:738
>     Yn http://localhost:1337/admin/main.58146bb6.js:735
>     qr http://localhost:1337/admin/main.58146bb6.js:735
>     Gr http://localhost:1337/admin/main.58146bb6.js:735
>     jr http://localhost:1337/admin/main.58146bb6.js:735
> main.58146bb6.js:1426:2446

**Comment by VincentDevp** [comment]
I met the same issue while create a field with datetime format, after click advanced settings, the web page change to blank

**Comment by Boegie19** [comment]
Duplicate of https://github.com/strapi/strapi/issues/18123
